### DISCUSSION TITLE
4.6.26

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#503] [Announcements]
#### ADDED
- 直播间添加开关底部礼物栏的按钮 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/488
- 支持设置新标签页到首页中
- 支持搜索已下载视频
- 支持搜索播放历史记录
#### CHANGED
- 美化直播弹幕栏 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/462
- 优化直播播放器加载行为 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/462
- 支持设置请求时使用的build值
#### FIXED
- 修复订阅合集视频在视频动态页中无法打开
- 修复港澳台番剧播放 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/470
- 修复从非番剧页打开番剧时产生的错误 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/472
- 直播间部分问题修复 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/462
- 画面尺寸添加兼容模式，修复部分设备缩放画面黑屏的问题 @JimHans  https://github.com/ywmoyue/biliuwp-lite/pull/475
- 修复创建收藏夹私密问题 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/481
- 修复热门页没有up主名称 @GD-Slime  https://github.com/ywmoyue/biliuwp-lite/pull/480
- 修复互动视频可能无选项问题
- 修复视频详情页视频标签不显示

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.26'>4.6.26</a>.</em>

**Comment by KennethWuWu** [comment]
大佬勤快6666
